templates/home/index.html

```diff
@@ -106,7 +106,7 @@ <h2>VISION</h2>
                 </div>
                 <div data-aos='fade-left'>
                     <h2>MISSION</h2>
-                    <p>Our mission is to erradicate illiteracy from India We are moving toward our mission by implementing this in IIIT Jabalpur by giving quality and value education to student of nearby villages. Our ultimate goal to extend our helping hand to each and every needy in India and make new India in the comming days.</p>
+                    <p>Our mission is to eradicate illiteracy from India. We are moving towards our mission by implementing this in IIIT Jabalpur by giving quality and value education to students of nearby villages. Our ultimate goal is to extend our helping hand to each and every needy in India and make a new India in the upcoming days.</p>
```

Comment:
I think `coming` will be more apt here. Also, it should be "implementing this in IIIT Jabalpur, giving quality and value education".

Reply:
> Thanks a lot @Saatvik21 for working on this. This was really important and you've done great work with it. I reviewed it and some little changes are needed at two more places. Please fix them.
> 
> **Tip:** You can make your changes, stage them and then instead of running `git commit ...`, run `git commit --amend`. It will make the changes in your last commit and you won't need to follow the squashing process again.
> 
> One last concern is that, the text in Vision section don't feel right (it doesn't make any sense). Do you or @tmo66 have any suggestion for this?

Yes sir, to be honest the vision section seems a bit same as the mission section also the last line doesn't make any sense. I would suggest that we can change it by writing something like "Our vision is to promote education as a vehicle for rural development and provide it to students who cannot otherwise access or afford it". 

Let me know if this is right and I should work on this.

Reply:
@Saatvik21 We'll finalize and let you know by tomorrow what should go in the Vision section.

Reply:
@Saatvik21 Our vision is to promote education as a vehicle for rural development and provide it to students who cannot otherwise access or afford it. By empowering children with education, we want to make sure that they live out their potential even under adverse circumstances. And light the future of generations to come.

Reply:
@garg3133 Sir, I just did `git commit --amend` after doing the above changes but the commit hasn't changed even after doing the changes. Do i also have to `git push origin 126` to push my changed commits?. And sir, it also says that my forked repo is 1 commit ahead, 3 commits behind, so should I first update my repo and then do the changes or directly push by `git push -f .....`

Reply:
You should force push your changes to Github: `git push origin +126`. No need to pull first.

Reply:
> You should force push your changes to Github: `git push origin +126`. No need to pull first.

Thank you sir, it's done!